packages/lib/markdownUtils.ts

```
@@ -224,17 +224,45 @@ const markdownUtils = {
+		let title = '';
+
+		for (const line of lines) {
+			const trimmed = line.trim();
+			if (trimmed) {
+				title = trimmed;
+				break;
+			}
+		}
+
+		title = title.replace(/<\/?(ins|del|mark|sub|sup)>/g, ' ');
+		title = title.replace(/!?\[([^\]]*)\]\(([^\)]*)\)/g, (_match, text, url) => {
```

ESLint

[error] Unnecessary escape character: \).

(no-useless-escape)